test/controllers/songs_controller_test.rb

```diff
@@ -35,11 +37,30 @@ class SongsControllerTest < ActionController::TestCase
     assert_includes(songs_data, songs(:glorious_day))
   end
 
-  test "should get new" do
+  # "new" action tests
+  test "new song page should load successfully" do
     get :new
     assert_response :success
   end
 
+  # "create" action tests
+  test "submitting a valid song should result in a new song in the database" do
```

Comment:
For this test, I would have loaded the new song from the database and assert it has the right values but unfortunately, probably due to a Minitest bug, `Song.find_by_name("New Song")` returned nothing after the post. I tried for like an hour to get it to work :(